app/src/main/java/edu/uc/groupproject/covid19tracker/service/GlobalDataProvider.kt

```diff
@@ -32,10 +32,10 @@ class GlobalDataProvider {
                      * create a json object of the response body string so that function can
                      * retrieve specific data from json
                      */
-                    var worldStat = response.body()!!.string()
-                    var Jobject = JSONObject(worldStat)
-                    var o = Jobject[dataType]
-                    var s = o as String
+                    val worldStat = response.body()!!.string()
+                    val Jobject = JSONObject(worldStat)
+                    val o = Jobject[dataType]
+                    val s = o as String
                     data.add(s)
```

Comment:
Changed as the variables remained constant so the variable would be better off as a val

Reply:
That's generally a good recommendation in Kotlin.  Declare as val by default, unless you believe the variable will be reassigned later.